docs/source/community/pull_requests.rst

```diff
@@ -0,0 +1,360 @@
+   writing comments like "should be capitalized".
+
+Reviewers of a PR are expected to:
+
+#. Understand what the PR is supposed to accomplish.
+#. If necessary, the reviewers should help the author refine the PR contents.
+
+   - Should the PR (and corresponding issue) be split into multiple issues/PRs?
+   - Did the author miss any obvious concerns?
+
+#. Keep an eye on the PR as it progresses. The frequency of "check-ins" should
+   be inversely proportional to the author's familiarity with the process,
+   *i.e.*, keep a closer eye on newer authors than seasoned veterans.
+#. Comment on the code when issues are spotted.
+
+   - Is the code using existing infrastructure to the extent possible?
+   - Is the code accruing technical debt?
+   - Is the formatting consistent? (Don't worry about formatting which CI will
+     fix)
+
+#. When the PR is marked as ready to go, complete a final pass through the code
+   flagging any potential issues.
+#. If issues arise work with the author to resolve them. Repeating the previous
+   step as necessary.
+#. If you are the last reviewer to approve a PR then merge it (assuming all
+   CI workflows have passed).
+
+Notes on PR Quality
+===================
+
+.. note::
+
+   The contents of this section are written assuming a 1.0 has occurred. We
+   admittedly have not lived up to the lofty standards of this section and
+   part of getting to a 1.0 is making sure existing code meets or exceeds
+   these standards.
+
+.. note::
+
+   Occasional contributors from outside the project are not the target of this
+   section. This section is targeting developers who are part of the team.
+
+The checklist on the PR template is admittedly asking a lot of the author.
+NWChemEx is designed to be a modular code. The vast majority of electronic
+structure development occurs in modules. Each of these modules are disjoint,
+and can be separately hosted. In turn, you can (and should) go through the
+"just get something working" phase outside of the NWChemEx repositories.
+
+By making a PR which will add a module into an NWX repository you are saying the
+module is ready for users. This does not necessarily mean that it is as
+performant as it is going to get, or that the module is fully featured. It
+simply means that the contents of the module have been vetted for correctness
+and the module has been found potentially useful. Once the module is merged it
+becomes available to users, and since taking it back would break any code that
+uses the module, we as a project are obligated to support that module or break
+API. The point being, we expect the module to be fully documented, tested, and
+to adhere to the organization's standards so that we can begin stewardship of
+the module.
+
+For PRs addressing infrastructure the requirements are a bit different.
+Since infrastructure exists outside modules, it can't really be staged until
+it is ready. For infrastructure we will allow some technical debt, as long as
```

Comment:
I am not quite sure what you mean by "it can't really be staged...".  Is it because it may cross over multiple repos? Maybe include an example or what you mean here.

Reply:
I took another stab at explaining what I meant. See if that's more clear.